package.json

```diff
@@ -18,7 +18,7 @@
     "test-cover": "babel-node node_modules/.bin/isparta cover --report text $npm_package_config_ISPARTA_OPTS node_modules/.bin/_mocha -- --require ./test/_setup.js 'test/**/*_test.js'",
     "test-cover-html": "babel-node node_modules/.bin/isparta cover --report html $npm_package_config_ISPARTA_OPTS node_modules/.bin/_mocha -- --require ./test/_setup.js 'test/**/*_test.js' && open coverage/index.html",
     "test-nocover": "node_modules/.bin/_mocha --compilers js:babel/register --require ./test/_setup.js 'test/**/*_test.js'",
-    "lint": "node_modules/.bin/eslint src test"
+    "lint": "eslint src test"
```

Comment:
npm adds "node_modules/.bin" to the $PATH automatically when running via `npm run`, so we can drop the extra path info here for better readability.